crawler/attribution/entrypoints/prototype-overload.js

```diff
@@ -0,0 +1,21 @@
+{
+    const descriptor = Object.getOwnPropertyDescriptor(
+        Document.prototype,
+        'createElement'
+    );
+    const origImpl = descriptor.value;
+    const src = document.currentScript.src;
+    descriptor.value = function () {
+        console.log(src, navigator.userAgent);
+        window.addResult('Navigator.prototype.userAgent', 'createElement prototype overload', src);
+        return origImpl.apply(this, arguments);
+    };
+
+    Object.defineProperty(
+        Document.prototype,
+        'createElement',
+        descriptor
+    );
+
+    document.createElement('div');
```

Comment:
This doesn't capture the intent of what I meant here. Another script would be calling this probably 1P.

Reply:
gotcha, I've added another case and moved most of the JS entry points to 3p

Reply:
This is the 1p hooked the method and the 3p called it; which is interesting (similar to the jQuery case we discussed).

I was hoping we'd cater for the other way around where 3p hooks the method and then 1p calls it.